src/main/java/com/fasterxml/jackson/databind/deser/CreatorProperty.java

```diff
@@ -354,4 +404,13 @@ private void _reportMissingSetter(JsonParser p, DeserializationContext ctxt) thr
             throw InvalidDefinitionException.from(p, msg, getType());
         }
     }
+
+    /**
+     * @since 2.18
+     */
+    public Map<Object, Object> initMap(DeserializationContext context, SettableAnyProperty anySetter)
```

Comment:
Should probably be renamed as something like "createAnyPropertyMap()" or such?
"init" would suggest initialization of `this` object which is not being done (was worried about that being the case from name).